lua/nvim-lsp-installer/servers/clangd/install.mjs

```diff
@@ -0,0 +1,20 @@
+const VERSION = "11.0.0";
```

Comment:
not sure actually needed, but no try catching and nice errors here? or that's too overkill?

Reply:
zx runs in "`set -e`" mode [by default](https://github.com/google/zx#prefix), so if a command fails its promise will just reject and abort the process. The Lua code that starts the script will then handle [handle errors](https://github.com/williamboman/nvim-lsp-installer/blob/zx/lua/nvim-lsp-installer/server.lua#L153)